RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -148,7 +148,16 @@ struct SubscriptionDetailsView: View {
             VStack(alignment: .leading) {
                 Text("\(subscriptionInformation.title)")
                     .font(.headline)
-                Text("\(subscriptionInformation.explanation)")
+
+                var explanation: String {
+                    return subscriptionInformation.active ? (
+                        subscriptionInformation.willRenew ?
+                            String(localized: "This is your subscription with the earliest billing date.") :
+                            String(localized: "This is your subscription with the earliest expiration date.")
+                    ) : String(localized: "This subscription has expired.")
```

Comment:
Afaik, the plan is to get all the strings we can from the backend, to make sure everything is using the same locale (considering the backend will only send a single locale). So I would probably move these strings to be obtained from the `Localization` struct in the customer center data instead. Happy to discuss it more over Slack though!

Reply:
Ah yes, that makes sense. I was definitely counting on this possibility. Will refactor.

Reply:
This is now done: https://github.com/RevenueCat/purchases-ios/pull/4083/commits/142ab1db2a822940a37d31366e495ed5b747ab80.